grpc-context-utils/src/main/java/org/hypertrace/core/grpcutils/context/RequestContext.java

```diff
@@ -138,4 +138,15 @@ public ContextualKey<Void> buildInternalContextualKey() {
   public <T> ContextualKey<T> buildInternalContextualKey(T data) {
     return new DefaultContextualKey<>(this, data, List.of(TENANT_ID_HEADER_KEY));
   }
+
+  @Override
+  public String toString() {
+    final String emptyValue = "{}";
+    return "RequestContext{"
+        + "headers="
+        + headers
+        + ", jwt="
+        + getJwt().map(Jwt::toString).orElse(emptyValue)
```

Comment:
a bit duplicative but I suppose it makes it easier

Reply:
I assumed we would want to see jwt details too